src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+/*
+Language: Linden Scripting Language
+Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+Author: Builder's Brewery <buildersbrewery@gmail.com>
+Category: scripting
+*/
+
+function(hljs) {
+
+    var LSL_STRING_ESCAPE_CHARS = {
+        className: 'subst',
+        begin: '\\[\\\\nt"\\]'
```

Comment:
LSL strings (with `\\`, `\"`, `\n` and `\t` as  **only** escape sequences) can contain JSON and HTML code.

---

Illegals are generally not highlighted in LSL. They can however be categorized into `invalid.broken`, `invalid.deprecated`, `invalid.illegal`, `invalid.unimplemented` as well as (admin only) `reserved.godmode`, `reserved.log`.